src/CONST.ts

```diff
@@ -2474,6 +2474,10 @@ const CONST = {
     get RESTRICTED_ACCOUNT_IDS() {
         return [this.ACCOUNT_ID.NOTIFICATIONS];
     },
+    // Account IDs that can't be added as a group member
+    get NON_ADDABLE_ACCOUNT_IDS() {
+        return [this.ACCOUNT_ID.NOTIFICATIONS, this.ACCOUNT_ID.CHRONOS];
+    },
```

Comment:
Based on this [comment](https://github.com/Expensify/App/issues/43841#issuecomment-2256829529), we don't want the add to group button to show for Chronos too, so I created a new list here.

But, I see that on the group invite list page, we exclude all expensify accounts, including Concierge, so searching for those accounts won't show up in the invitation search list.
https://github.com/Expensify/App/blob/685bb4c7bbaa2ef332896de5cace193edb9c6a51/src/pages/InviteReportParticipantsPage.tsx#L52-L53

Should we hide the add to group button for all expensify accounts?

@iwiznia @trjExpensify @abdulrahuman5196

Reply:
Historically, people have been able to have a groupDM with Concierge for support. It's the same as emailing concierge@expensify.com with additional email addresses. That said, now we have transitioned to "Groups" where membership isn't fixed on creation (meaning, everyone else in the group can leave freely), does that cause problems and someone can end up with two Concierge DMs if that happens? 🤔 CC: @marcaaron @JmillsExpensify

Reply:
We ended up covering this case in the Group Chats design doc. The intended design spec should work like this.

> A goal of this doc is to completely deprecate Group DMs in favor of Group Chats. In doing that, the behavior of chatting with Concierge via email will change a bit. When a user emails Concierge and CC’s another user we will create a new Group Chat instead of trying to locate an existing chat between the participants. The consequence is that some historical chat messages between these participants can exist across several different reports vs. just one. This is a case we can potentially optimize for in the future, but decided to completely deprecate the Group DM pattern for now.

Reply:
Got it, so in product you should be able to `Add to group` Concierge still? If everyone else is removed/leaves, it'll still be a "Group" technically and not be confused with the Concierge DM?

Reply:
> If everyone else is removed/leaves, it'll still be a "Group" technically and not be confused with the Concierge DM?

Removing all group members won't convert it to a DM. Leaving the group chat will remove the report.

Based on the design spec, then we should be able to add a concierge to the group. We can update the non-addable to group account list later if needed.

Reply:
Yeah basically Group Chats are not defined by the number or makeup of their members. You can have a Group Chat with yourself as the only user. You can have a Group Chat with you and Concierge. You can remove Concierge from that chat later if you want. The 1:1 DM with Concierge will always be a 1:1 DM with Concierge though and you should not have more than 1 of those. Hope that makes sense!

Reply:
Great, thanks for the confirmation!

@abdulrahuman5196 we can start the review here then